cypress/integration/template-starter/channels.js

```diff
@@ -2,7 +2,9 @@ import { URL_EXAMPLES_STARTER_CHANNELS } from '../../support/urls';
 
 describe('Channels', () => {
   it('should render page', () => {
-    cy.login({ redirectToUri: URL_EXAMPLES_STARTER_CHANNELS });
+    cy.login({
+      redirectToUri: URL_EXAMPLES_STARTER_CHANNELS,
+    });
```

Comment:
The e2e test output has changed, but the output as it now stands (expanding the menu and activating the current menu item if the screen width is greater than 1024px) has been approved by UX, so I believe the chages in these two tests should be accepted.